Cargo.lock

```diff
@@ -502,6 +503,23 @@ dependencies = [
  "url 2.1.1",
 ]
 
+[[package]]
+name = "deno_lint"
+version = "0.1.6"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "98a1423991b250b313ea7d2cbe6b04dbf1235cd531ccaec32912e68fa41e1f0b"
+dependencies = [
+ "clap",
```

Comment:
Can you make it so deno_lint does not depend on clap?

Reply:
Done